File: src/app/community/scratch/clean_community.py
import os

file_path = r'd:\Workstation DP-2\DP Clients\WEB DEVELOPMENT\Checkout\src\app\community\page.tsx'

with open(file_path, 'r', encoding='utf-8') as f:
    content = f.read()

# Replace literal backslash-quote with quote
content = content.replace(r'\"', '"')

# Fix the Expert key
content = content.replace('key={Expert-${i}}', 'key={`Expert-${i}`}')

with open(file_path, 'w', encoding='utf-8') as f:
    f.write(content)

print("File cleaned successfully.")

File: src/app/communities/community/scratch/clean_community.js
const fs = require('fs');
const path = require('path');

const filePath = path.join('d:', 'Workstation DP-2', 'DP Clients', 'WEB DEVELOPMENT', 'Checkout', 'src', 'app', 'community', 'page.tsx');

let content = fs.readFileSync(filePath, 'utf8');

// Replace literal backslash-quote with quote
content = content.replace(/\\"/g, '"');

// Fix the Expert key
content = content.replace('key={Expert-${i}}', 'key={`Expert-${i}`}');

fs.writeFileSync(filePath, content);
console.log("File cleaned successfully.");
